ngraph_bridge/enable_variable_ops/ngraph_rewrite_for_tracking.cc

```diff
@@ -46,25 +73,22 @@ Status RewriteForTracking(Graph* graph, int graph_id) {
   for (auto node : graph->op_nodes()) {
     auto itr = REWRITE_REPLACE_OP_MAP.find(node->type_string());
     if (itr != REWRITE_REPLACE_OP_MAP.end()) {
-      NGRAPH_VLOG(1) << "Checking: " << DebugNode(node) << " " << node->name();
-
-      bool outputs_ng_supported = true;
+      NGRAPH_VLOG(5) << "Checking: " << DebugNode(node) << " " << node->name();
+
+      // Check if the TF Tensor of the Variable needs to be updated
+      // Update is required in two cases
+      // 1. If any of the outputs of this Op is feeding into a TF Op
+      // 2. If this is a static input to NGraphEncapsulate
```

Comment:
We can add some more comments here. In case of static input, the Compute fn will read the value from TF. which means if TF is not updated it will get stale values from TF etc.